modules/ui/src/main/scala/lucuma/ui/refined/package.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.char.UpperCase
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.api.RefinedTypeOps
+import eu.timepit.refined.collection.Forall
+import eu.timepit.refined.boolean.And
+
+/**
+ * Convenience refined definitions.
+ */
+package object refined {
+  type UpperNES = String Refined And[NonEmpty, Forall[UpperCase]]
```

Comment:
Nice

Reply:
Note that this is for alpha characters, not spaces, number, etc. To enforce what we think more generally think of as an uppercase string, I think we could use `Forall[Not[LowerCase]]`

Reply:
Good catch! Thank you! I already merged but I will make another PR with this change.